03-transactions.md

```diff
@@ -442,13 +514,16 @@ For efficiency, keys are generated from a series of per-commitment secrets
 that are generated from a single seed, which allows the receiver to compactly
 store them (see [below](#efficient-per-commitment-secret-storage)).
 
-### `localpubkey`, `local_htlcpubkey`, `remote_htlcpubkey`, `local_delayedpubkey`, and `remote_delayedpubkey` Derivation
+### `localpubkey`, `local_htlcpubkey`, `remote_htlcpubkey`, `local_delayedpubkey`, `remotepubkey` and `remote_delayedpubkey` Derivation
 
-These pubkeys are simply generated by addition from their base points:
+These pubkeys are simply generated by addition from their base points. For newer
```

Comment:
Assuming a privacy third-party watchtower, does anyone has a sketch of the encrypted blobs sent for each update and the scope of services (just punishment or also timing-out HTLCs...) ? It may help to anchor the discussion on this particular spec change (or at least we conscientiously diminish watchtowers future privacy expectations)

Reply:
@rustyrussell gave more context on the watchtower implications in https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-October/002262.html

Conclusion seems to be to not remove the key rotation. Open question is still whether the anchor key should be rotated or not.

Reply:
Updated PR to leave key rotation unchanged. For now assumed anchor key is not rotated.